v2/cmd/controller/main.go

```diff
@@ -39,7 +39,7 @@ func main() {
 
 	var metricsAddr string
 	var enableLeaderElection bool
-	flagSet.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
+	flagSet.StringVar(&metricsAddr, "metrics-addr", "0", "The address the metric endpoint binds to.")
```

Comment:
The default of `0` doesn't seem to make sense? Maybe `:8080` didn't either? does 0 here mean "don't run metrics at all"? That's sorta surprising because it's a string so I'd expect that empty string did that?

Reply:
Yes, you are right. "0" is for disabling the metrics (standard prometheus setting). 

I agree on this being a bit unambiguous, should add a flag `enable-metrics` which then forces `metricsAddr` to be "0"?